source/_components/googlehome.markdown

```diff
@@ -0,0 +1,69 @@
+---
+layout: page
+title: "Google Home"
+description: "Instructions on how to connect to your Google Home device."
+date: 2018-11-04 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: google_home.png
+ha_release: 0.83
```

Comment:
0.88

Reply:
The component is new but not the platform. 🤷‍♂️

Reply:
Was unsure about this, updated to 88